packages/block-library/src/list/test/__snapshots__/index.js.snap

```diff
@@ -19,9 +19,11 @@ exports[`core/list block edit matches snapshot 1`] = `
           data-is-placeholder-visible="true"
           role="textbox"
         >
-          <br
-            data-mce-bogus="1"
-          />
+          <li>
```

Comment:
Are these changes here normal?

Reply:
Yes, it was actually wrong before. We're expecting a padded multiline element.